addons/hr_fleet/views/employee_views.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_employee_form" model="ir.ui.view">
+        <field name="name">hr.employee.form.inherit.hr.fleet</field>
+        <field name="model">hr.employee</field>
+        <field name="inherit_id" ref="hr.view_employee_form" />
+        <field name="arch" type="xml">
+             <xpath expr="//sheet" position="before">
+                <header>
+                    <button name="action_get_claim_report" string="Claim Report"
+                    type="object" groups="fleet.fleet_group_manager"
+                    attrs="{'invisible': [('employee_cars_count','=', 0)]}"
```

Comment:
are you sur it work ?, because `employee_cars_count` is not prevent in the view 
`<field name='employee_cars_count' invisible="1"/>`

Reply:
@fmdl If you look, the field is present inside the second button ;)